modules/ezcontent_paragraphs/modules/ezcontent_card/config/install/core.entity_form_display.paragraph.card.default.yml

```diff
@@ -1,34 +1,53 @@
+uuid: 6ac74444-5b38-4e91-9d1a-d95e58a1c844
 langcode: en
 status: true
 dependencies:
   config:
     - entity_browser.browser.image_browser
+    - field.field.paragraph.card.field_background_color
     - field.field.paragraph.card.field_link
     - field.field.paragraph.card.field_media
     - field.field.paragraph.card.field_short_title
     - field.field.paragraph.card.field_subhead
     - field.field.paragraph.card.field_summary
+    - field.field.paragraph.card.field_text_position
     - field.field.paragraph.card.field_title
     - field.field.paragraph.card.layout_selection
     - paragraphs.paragraphs_type.card
   module:
+    - color_field
     - entity_browser
     - link
+    - text
+_core:
+  default_config_hash: gz6zqzTPcoL6zx_KrCx1hChc8TQSeOMvLgOYVkjvx-w
 id: paragraph.card.default
 targetEntityType: paragraph
 bundle: card
 mode: default
 content:
+  field_background_color:
```

Comment:
This field name creates ambiguity. Lets keep the name as `field_text_background_color`